### PR TITLE
Module1:Review your project from trials

### PR DESCRIPTION
**Pull request by hammaazarok**
-add preview project and setup linter
-edit HTML page
-edit style.css file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Miroverse Inc.
+- Qalb-E-Al
+
```
Comment (hammaazarok) [inline README.md:50]: [OPTIONAL] - [ ] you can add the Acknowledge to the freeCodeCamp site as you used their resources while your project.

**Review by hammaazarok — COMMENTED** [review]
Hi @hammaazarok

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Good commit messages
✅ Good READMe template
✅ Great pull request summary

Required Changes ♻️
Check the comments under the review
___________________________________________________________________________
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Tribute Page</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
```
Comment (hammaazarok) [inline index.html:13]: - [ ] kindly remove this link as it is not used anymore!

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
-# stephen-tribute-page
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```
Comment (hammaazarok) [inline README.md:2]: [OPTIONAL] 

- [ ] you can add a screenshot of your webpage to let know your visitors what the page looks like.

File: index.html (hunk 2 above)
Comment (muneebulrehman) [reply index.html:13]: @hammaazarok, Your comment should always sound like a suggestion. Remember, we don't want to push a narrative, we are just giving our opinion based on our experience and knowledge.

**Review by muneebulrehman — APPROVED** [review]
Hi @hammaazarok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

## Highlights
- Highlighted the code about the project.
- Raised optional suggestions as Optional.
- Used comments very well

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**